hack/run-in-container.sh

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Isolated container environment for development.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+CLEANUP=${CLEANUP:-} # if set, cleaning up local volumes
+DOCKER_LIB_VOLUME=${DOCKER_LIB_VOLUME:-benchmark-operator-docker-lib}
+DOCKER_GRAPH_VOLUME=${DOCKER_GRAPH_VOLUME:-benchmark-operator-docker-graph}
+DOCKER_GO_VOLUME=${DOCKER_GO_VOLUME:-benchmark-operator-go}
+NAME=${NAME:-benchmark-operator-dev}
+
+function usage() {
+    cat <<'EOF'
+This script is entrypoint to start a isolated container environment for development.
+
+Usage: hack/run-in-container.sh [-h] [command]
+
+    -h      show this message and exit
+
+Environments:
+
+    CLEANUP             if passed, clean up local caches
+    DOCKER_LIB_VOLUME   the name of docker lib volume, defaults: benchmark-operator-docker-lib
+    DOCKER_GRAPH_VOLUME the name of docker graph volume, defaults: benchmark-operator-docker-graph
+    DOCKER_GO_VOLUME    the name of go cache volume, defaults: benchmark-operator-go
+    NAME                the name of container
+
+Examples:
```

Comment:
What purpose is `run-in-container` script for?

Reply:
Using an isolated container environment for development makes it easier to do cleanup jobs when the environment messed up. And some tools used in the developing environemt might have different version in the developer's operating system, which might cause problems and confusion.
After all, using this script is optional, if anyone thinks it is not necessary, he can simply ignore this.

Reply:
It's ok to me.